general-remarks.md

```diff
@@ -136,11 +134,11 @@ Or
 
 ### Integration setup
 
-1. Obtain **Client Token** from Mews, which will identify the channel manager as a Mews client in the requests to the Mews API. The token is same for all connected hotels via the channel manager.
+1. Obtain **Client Token** from Mews, which will identify the channel manager as a Mews client in the requests to the Mews API. The token is same for all connected hotels via the channel manager per environment.
 2. Setup **Client Token** for Mews, which will identify Mews as a client of the channel manager in the request to the Channel Manager API. It can be same token \(recommended, simpler\) or it can be different one.
 3. Provide URL addresses of the Channel Manager API operations.
    * When implemening the Channel Manager API side, you can provide the endpoint URLs one by one.
-   * Production URLs can be provided after certification is completed. Ideally production environment should differ from test evironment.
+   * Production URLs can be provided after certification is completed. Ideally production environment should differ from test evironment because same URLs cannot be used across different environements. A separate set of URLs allows to keep demo environment for tests and enhancements.
```

Comment:
Why cannot same URLs work? Also, we don't have test env, we have only demo. If possible, I would drop the requirement that URLs have to differ, if they distinguish environment differently and have it on same domain, that's up to them.

Reply:
I am remember correctly someone had only 1 environment and there werent able to distinguish properly so then demo tokens were coming to production

Reply:
I also believe it's not possible technically to save same endpoints in prod and demo. So the question is if we want to invest time to approve this to allow partners to use the same URLs

Reply:
Yeah, but it's their problem. If there is a partner who can have one URL for all envs and internally performs HTTP routing based on "Mews client token", then we shouldn't force them to have different URLs. Anyway, since the URLs are setup in two different instances of Mews, we are even unable to force this or check this systematically.

Reply:
The routing is good argument to not force them to have different URLs as i remember travelclick and some custom do routing based on soap header so i woudnt force them

Reply:
Tested and now it's possible to save the same endpoints in different environments, so removing the sentence about URLs.